x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/dimension_container.scss

```diff
@@ -1,48 +1,3 @@
-@import '@elastic/eui/src/components/flyout/variables';
-@import '@elastic/eui/src/components/flyout/mixins';
-
-.lnsDimensionContainer {
-  // Use the EuiFlyout style
-  @include euiFlyout;
-  // But with custom positioning to keep it within the sidebar contents
-  animation: euiFlyout $euiAnimSpeedNormal $euiAnimSlightResistance;
-  left: 0;
-  max-width: none !important;
-  z-index: $euiZContentMenu;
-
-  @include euiBreakpoint('l', 'xl') {
-    height: 100% !important;
-    position: absolute;
-    top: 0 !important;
-  }
-
-  .lnsFrameLayout__sidebar-isFullscreen & {
-    border-left: $euiBorderThin; // Force border regardless of theme in fullscreen
-    box-shadow: none;
-  }
-}
-
-.lnsDimensionContainer__header {
-  padding: $euiSize;
-
-  .lnsFrameLayout__sidebar-isFullscreen & {
-    display: none;
-  }
-}
-
-.lnsDimensionContainer__content {
-  @include euiYScroll;
-  flex: 1;
-}
-
-.lnsDimensionContainer__footer {
-  padding: $euiSize;
-
-  .lnsFrameLayout__sidebar-isFullscreen & {
-    display: none;
-  }
-}
-
 .lnsBody--overflowHidden {
   overflow: hidden;
 }
```

Comment:
Was it intentional to leave this `.lnsBody--overflowHidden` style here? Asking because it looks like it was also copy/pasted to `flyout_container.scss`, meaning the same style is being applied twice. 

Does it belong here, in `flyout_container.scss`, or elsewhere? Seems an odd inclusion for `dimension_container.scss` or `flyout_container.scss`, but I don't know the full history.

Reply:
Removed the duplicate. That was an unintentional left over during the move